api/lib/domain/services/authentication/oidc-authentication-service.js

```diff
@@ -66,25 +71,25 @@ class OidcAuthenticationService {
       redirect_uri: redirectUri,
     };
 
-    const response = await httpAgent.post({
+    const httpResponse = await httpAgent.post({
       url: this.tokenUrl,
       payload: querystring.stringify(data),
       headers: { 'content-type': 'application/x-www-form-urlencoded' },
       timeout: settings.partner.fetchTimeOut,
     });
 
-    if (!response.isSuccessful) {
+    if (!httpResponse.isSuccessful) {
       const message = 'Erreur lors de la récupération des tokens du partenaire.';
-      const dataToLog = httpErrorsHelper.serializeHttpErrorResponse(response, message);
+      const dataToLog = httpErrorsHelper.serializeHttpErrorResponse(httpResponse, message);
       monitoringTools.logErrorWithCorrelationIds({ message: dataToLog });
```

Comment:
```suggestion
      monitoringTools.logErrorWithCorrelationIds({
        context: {
          identityProvider: this.identityProvider,
          organizationName: this.organizationName,
        },
        'feature-team': 'access',
        message: dataToLog,
      });
```

Comme tu l'as ajouté en remarque, je pense que tu pourrais déjà le mettre en place. `feature-team` serait la clé sur laquelle on pourra filtrer et on pourrait même dire que cette clé serait globale à toutes les équipes Pix (à voir avec les captains).

Reply:
Je propose qu'on réfléchisse avant à ce qu'on veut monitorer et ce que ça va apporter de plus.
On est déjà capable de filtrer par idp car les url qu'on appelle ne sont pas les mêmes.
Mettre de nouvelles datas pour de nouvelle datas sans usage derrière, je vois moyennement l'intérêt